<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VisiTHCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QWidget" name="layoutWidget0" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>249</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="simulateButton" >
       <property name="text" >
        <string>Simulate X Days</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="smokeCheckBox" >
       <property name="text" >
        <string>Smoke?</string>
       </property>
       <property name="tristate" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="simulateSpinBox" />
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget1" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>10</y>
      <width>219</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="enterContentButton" >
       <property name="text" >
        <string>Manually Enter Content</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="contentSpinBox" >
       <property name="maximum" >
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>196</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="smokeButton" >
       <property name="text" >
        <string>Smoke Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nonSmokeButton" >
       <property name="text" >
        <string>Non-Smoke Day</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget3" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>170</y>
      <width>258</width>
      <height>217</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="HistoryLabel" >
       <property name="text" >
        <string>History</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="historyListWidget" />
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>310</x>
      <y>0</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title" >
     <string>Status</string>
    </property>
    <widget class="QWidget" name="layoutWidget4" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>177</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="thLabel" >
            <property name="text" >
             <string>THC Percentage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="percentLcdNumber" />
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="dayPassLabel" >
            <property name="text" >
             <string>Days Till Pass</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="passLcdNumber" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="dayLabel" >
          <property name="text" >
           <string>Day</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="dayLcdNumber" >
          <property name="value" stdset="0" >
           <double>1.000000000000000</double>
          </property>
          <property name="intValue" stdset="0" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="clearButton" >
    <property name="geometry" >
     <rect>
      <x>390</x>
      <y>130</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget5" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>130</y>
      <width>360</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="daysTillButton" >
         <property name="text" >
          <string>Days Till X Percentage</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="tillSmokeCheckBox" >
         <property name="text" >
          <string>Smoke?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="daysTillSpinBox" >
         <property name="maximum" >
          <number>999</number>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLCDNumber" name="daysTillLcdNumber" />
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
